redisenterprise/assets/dashboards/redisenterprise_rof.json

```
@@ -0,0 +1,513 @@
+{
+    "title": "Enterprise Redis On Flash",
```

```suggestion
    "title": "Redis Enterprise On Flash",
```
Nit: for consistency with the other dashboards